src/mavlink-router/endpoint.cpp

```diff
@@ -707,9 +707,34 @@ int UartEndpoint::add_speeds(std::vector<unsigned long> bauds)
 }
 
 UdpEndpoint::UdpEndpoint()
```

Comment:
packet coalescing with delays should probably only apply to really "outgoing" network

we are using udp also for internal traffic (yes we should not, that should all be UDS), but for the time being internal traffic should not be subject to delays

Reply:
Agreed. Before merging this I'd like to add the capability of configuring the value per-endpoint. For radio links, having a delay of less than ~20ms probably won't help that much for typical traffic. For internal of course we don't want any delay at all.